src/Components/DrillIllustration.js

```diff
@@ -88,7 +147,7 @@ const DrillIllustration = props => {
   const displayAnimation = ({ illustrationSource, instruction, title }) => {
     return (
       <>
-        <Animation readonly widthRatio={1} heightRatio={1 / 2} animation={illustrationSource} />
```

Comment:
You rolled back my PR ? bad merge ?

Reply:
Je pense que je fais de la me*** avec mes PR. Désolé. Je fais n'importe quoi sur les merges

Reply:
Pas de soucis, c'est toujours un peu chiant